src/screens/ContactScreen/index.js

```diff
@@ -262,9 +280,8 @@ const styles = StyleSheet.create({
   containerStyle: {
     marginHorizontal: 16
   },
-  subtitleStyle: (selectedUsers) => ({
-    color: selectedUsers.length > 0 ? COLORS.anon_primary : COLORS.gray4,
-    marginEnd: 8
+  subtitleStyle: (sourceScreen) => ({
+    color: sourceScreen === ANONYMOUS ? COLORS.anon_primary : COLORS.signed_primary
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The subtitle style has been updated to handle different scenarios based on the source screen. If the source screen is `ANONYMOUS`, a specific color is used; otherwise, a different color is used. This change enhances the flexibility of the UI customization process.
```diff
+ subtitleStyle: (sourceScreen) => ({
+   color: sourceScreen === ANONYMOUS ? COLORS.anon_primary : COLORS.signed_primary
+ })
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->